tensorboard/backend/application.py

```diff
@@ -277,6 +277,9 @@ def _serve_plugins_listing(self, request):
     """
     response = collections.OrderedDict()
     for plugin in self._plugins:
+      if type(plugin) is core_plugin.CorePlugin:  # pylint: disable=unidiomatic-typecheck
```

Comment:
Optional, but we could just do `isinstance()` - CorePlugin won't ever have subclasses but I think one could argue that if it did then it makes sense to skip those too.

Reply:
The existing uses of `type(—) is —` were intentional (see #2006). For
now, I’ll keep it consistent with the other two in the file, and we can
change them all together if we so choose.

Reply:
Oh right, I forgot about that.  Ok, sure.